src/rollup/types.d.ts

```diff
@@ -487,8 +491,10 @@ export interface SerializedTimings {
 }
 
 export interface OutputAsset {
+	type: 'asset';
```

Comment:
The properties should be ordered alphabetically. I do not know why TSLint does not seem to pick it up in this file (it is flagged in other files), but my IDE at least flags it. The same goes for the `OutputChunk` interface.